components/storage/ComparisonCards.vue

```diff
@@ -23,3 +55,9 @@ export default {
   },
 };
 </script>
+
+<style>
+ul {
+  text-align: left;
```

Comment:
instead of using css here, use `has-text-left` ([docs](https://bulma.io/documentation/helpers/typography-helpers/#alignment)) on the `ul` tag